### ISSUE TITLE
Dependencies are not up-to-date.

### ISSUE THREAD

**Issue by cameron-martin** [Twey/gloss-banana#6]
The `gloss` and `reactive-banana` dependencies are both out of date.

Would it be possible to update these?

**Comment by Twey** [comment]
Yes, but it's non-trivial since the `reactive-banana` API has changed.  Will get to it some time this week (or, patches welcome).

**Comment by cameron-martin** [comment]
I'll give it a go over the next few days and submit a PR, but my familiarity with reactive banana, and Haskell in general, is not great so I can't make any promises.

**Comment by cameron-martin** [comment]
I am not sure what the code commented as "make sure the Behavior doesn't leak memory if mPicture ignores one or both kind of events" is achieving. Would it be possible to explain this?

**Comment by cameron-martin** [comment]
Any update on this?
